exercises/word-count/src/example.c

```diff
@@ -0,0 +1,75 @@
+#include <stdlib.h>
+#include <string.h>
+#include <ctype.h>
+#include "word_count.h"
+
+// checks to see if we are counting a given word:
+//    if word counted already it returns index.
+//    elseo returns -1
+static int word_match(char *testWord, int current_word_count, WordCount_Word_t words[MAX_WORDS])
```

Comment:
My first instinct was to suggest just using the more common form of `WordCount_Word_t *words`.  But I think this would actually be a good lesson for those who don't know that `[]` are dereference operators.